arrow-fx-android/build.gradle

```diff
@@ -0,0 +1,54 @@
+buildscript {
+    repositories {
+        google()
+    }
+    dependencies {
+        classpath("com.android.tools.build:gradle:3.6.0")
```

Comment:
This can be 4.0.0 now. https://twitter.com/AndroidDev/status/1266054894043058176?s=19